webapp/lib/app/nook/view.dart

```diff
@@ -594,6 +605,8 @@ class MessageView {
       ..append(_message);
 
     _messageBubble = new DivElement()
+      ..dataset['conversationId'] = conversationId
+      ..dataset['messageId'] = messageId
```

Comment:
nit - we can use the same format as the others:
```suggestion
      ..dataset['conversation-id'] = conversationId
      ..dataset['message-id'] = messageId
```